datafusion/sqllogictest/test_files/common_subexpr_eliminate.slt

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#############
+## Common Subexpr Eliminate Tests
+#############
+
+statement ok
```

Comment:
I recommend we move this test out of its own .slt file and into https://github.com/apache/arrow-datafusion/blob/main/datafusion/sqllogictest/test_files/expr.slt so it is eaiser to find

Reply:
done.